orders/src/main/java/com/lambdaschool/orders/model/Orders.java

```diff
@@ -0,0 +1,104 @@
+package com.lambdaschool.orders.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "orders")
+public class Orders
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long ordnum;
+
+    private double ordamount;
+    private double advanceamount;
+    private String ordersescription;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "custcode",
+            nullable = false)
+    @JsonIgnore
+    private Customers customer;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "agentcode",
+            nullable = false)
+    @JsonIgnore
+    private Agents agent;
```

Comment:
Line 28 - Don't rely on `@JsonIgnore` - it will prevent printing of an order's agent's information if you print some order object(s) directly.

Also, since you're using `fetchType.LAZY` here, make sure to ignore the property `hibernateLazyInitializer` as well to prevent its information from printing unnecessarily.

Use `@JsonIgnoreProperties`, and pass in String values representing the agent object's properties `({"customers", "orders", "hibernateLazyInitializer"})`.